cfme/tests/infrastructure/test_vm_engine_relationship.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.provider(classes=[InfraProvider], selector=ONE_PER_TYPE),
+    pytest.mark.usefixtures('setup_provider'),
+    test_requirements.rhev,
+]
+
+
+@pytest.fixture()
+def new_vm(provider):
+    vm = provider.appliance.collections.infra_vms.instantiate(
+        random_vm_name('engine-rel', max_length=16), provider
+    )
+    vm.create_on_provider(find_in_cfme=True)
+    yield vm
+    vm.cleanup_on_provider()
+
+
+def mark_vm_engine_relationship(vm, relationship):
```

Comment:
Worth adding as a method to InfraVM?

Reply:
@mshriver good idea!

Reply:
oh, it was almost there, I'm just adding one more method :)

Reply:
Looks like the functionality was replaced, but method left in place?

Reply:
oops, true!